scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -540,6 +540,9 @@ task CreateManifestAndOptionallyCopyOutputs {
     declare -a output_vcf_bytes=(~{sep=' ' output_vcf_bytes})
     declare -a output_vcf_index_bytes=(~{sep=' ' output_vcf_index_bytes})
 
+    # (Possibly) create a manifest of VCFs and indexes to bulk copy with `gcloud storage cp`.
+    echo -n > vcf_manifest.txt
```

Comment:
```suggestion
    touch vcf_manifest.txt
```

Reply:
I prefer the former because if (and in this case it's not going to happen) vcf_manifest.txt exists all you have done is updated it's time stamp. The former creates a zero length file.